api/test/wfl/integration/executor_test.clj

```diff
@@ -482,11 +485,11 @@
                   "Method configuration data reference name should be updated")
               (is (== inc'dMcVersion (:methodConfigVersion mc))
                   "Method configuration's actual version should be incremented")
-              (is (= rootEntityTypePost (:rootEntityType mc))
+              (is (= snapshot-table-name (:rootEntityType mc))
                   "Method configuration root entity type should be snapshot's table name"))]
       (with-redefs-fn
         {#'firecloud/method-configuration        firecloud-method-configuration
-         #'datarepo/snapshot                     datarepo-snapshot
+         #'datarepo/snapshot                     mock-datarepo-snapshot
          #'firecloud/update-method-configuration verify-firecloud-update-params}
         #(#'executor/update-method-configuration! executor reference))
       (let [reloaded (reload-terra-executor executor)]
```

Comment:
When I see mocks stacked 8 deep I wonder what is left to actually test.
(Unfortunately I have no better idea of what to do now.)

Reply:
I am in the same boat.  When I add new tests these days, I'm more inclined to unit over integration for this reason, but that's me kicking the can.